packages/app/src/resource/QuestionnaireBotsPage.tsx

```diff
@@ -82,31 +83,47 @@ export function QuestionnaireBotsPage(): JSX.Element {
         <InputLabel size="lg" htmlFor="bot">
           Connect to bot
         </InputLabel>
-        <Group>
-          <ResourceInput name="bot" resourceType="Bot" onChange={(r) => setConnectBot(r as Bot)} />
-          <Button onClick={connectToBot}>Connect</Button>
-        </Group>
-        <Group>
-          <NativeSelect
-            name="subscription-trigger-event"
-            defaultValue="Create Only"
-            label="Subscription Trigger Event"
-            data={SUBSCRIPTION_INTERACTION_KEYS}
-            onChange={(event) =>
-              setSupportedInteraction(SUBSCRIPTION_INTERACTION_MAP[event.target.value as SubscriptionInteractionKey])
-            }
-          />
-        </Group>
+        {!questionnaire?.url ? (
+          <p>Cannot create new bot subscriptions until a canonical URL is added to the questionnaire.</p>
```

Comment:
Do we want to be this assertive?  We could easily support the existing behavior.

I'm a little bit nervous about the added friction of requiring people to come up with `Questionnaire.url` strings for this.